### ISSUE TITLE
Application wide -> alert user if required field isn't filled

### ISSUE THREAD

**Issue by dellch** [FLMNH-Informatics/regnum-prod#56]
This would need to be done on "submission" not while working... Unless we change the system to not save automatically with every change

**Comment by ncellinese** [comment]
I agree, let's generate an alert after they push the submission button
